packages/unit-testing/stubs/cache/CacheInstance.js

```diff
@@ -0,0 +1,82 @@
+define([], function () {
+    /**
+     * @class Cache
+     * @classdesc Named Cache for caching the result of expensive (in terms of time or governance) computations.
+     * @protected
+     * @constructor
+     */
+     function Cache() {
```

Comment:
Mising properties: this.name and this.scope
Acording to the documentation: https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_4642573343.html
the following functions are not present: this.toJSON and this.toString I think we should remove it.

The missing parts are present in Webstorm SS2 library file, cache.js.

Reply:
Added properties, as for the toJSON and toString I wouldn't worry too much, they are present in most of the instances (e.g. SearchInstance) and should not cause any problems for unit testing

Reply:
Later I read somewhere in the SuiteScript 2.X documentation that toString and toJSON are present in all the modules.